src/Extension/CoreExtension.php

```diff
@@ -1285,6 +1287,56 @@ public static function testEmpty($value): bool
         return '' === $value || false === $value || null === $value || [] === $value;
     }
 
+    /**
+     * Checks if a variable is a sequence.
+     *
+     *    {# evaluates to true if the foo variable is a sequence #}
+     *    {% if foo is sequence %}
+     *        {# ... #}
+     *    {% endif %}
+     *
+     * @param mixed $value A variable
```

Comment:
```suggestion
     * @param mixed $value
```

Reply:
Done in https://github.com/twigphp/Twig/pull/3859/commits/ee5dcd6444218f6640825c73266f124c9013f3fe